web_client/src/Passage.jsx

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react';
+import readPassage from './api';
+import './Passage.css';
+
+class Passage extends Component {
+    componentWillMount() {
+        this.setState({passageText: ''});
+        readPassage(this.props.finneganForeverUrl)
```

Comment:
`readPassage` should know about `finneganForeverUrl` so this component doesn't have to